handsontable/src/__tests__/core/settings.types.ts

```diff
@@ -610,6 +619,7 @@ const allSettings: Required<Handsontable.GridSettings> = {
   modifyFocusedElement: (row, column, focusedElement) => document.createElement('TD'),
   modifyData: () => {},
   modifyFocusOnTabNavigation: (tabActivationDir, visualCoords) => {},
+  tableFocusExit: (exitDirection) => {},
```

Comment:
Can you also add TS tests for all other new hooks that the PR adds?

Reply:
I found that the `tableFocusExit` does [not have](https://github.com/handsontable/handsontable/blob/feature/dev-issue-2707/handsontable/src/dataMap/metaManager/metaSchema.js) documentation. Can I ask you to add one?